easybuild/easyblocks/generic/intelbase.py

```diff
@@ -102,10 +105,41 @@ def extra_options(extra_vars=None):
             # disables TMP_PATH env and command line option
             'usetmppath': [False, "Use temporary path for installation", CUSTOM],
             'm32': [False, "Enable 32-bit toolchain", CUSTOM],
+            'components': [[], "List of components to install", CUSTOM],
         })
 
         return extra_vars
 
+    def parse_components_list(self):
+        """parse the regex in the components extra_options and select the matching components
+        from the mediaconfig.xml file in the install dir"""
+
+        mediaconfigpath = os.path.join(self.cfg['start_dir'], 'pset', 'mediaconfig.xml')
+        if not os.path.isfile(mediaconfigpath):
+            self.log.warning("Could not find pset/mediaconfig.xml in the build directory. Skipping list of components to install")
```

Comment:
if `parse_components_list` is only called in case `components` is specified, make this a `raise`